utils/conda_upload.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -x
+set -e
+
+die() {
+    echo >&2 "$@"
+    exit 1
+}
+
+# check input
+CORE=${CORE_DIR?"Environment variable CORE_DIR must be set to the cyclus repository directory."}
```

Comment:
Also, I'd prefer if this was named "CYCLUS" rather than core, to be consistent with the project name.